conda/shell/Scripts/activate.bat

```diff
@@ -15,14 +15,7 @@
 @set _args1_last=%_args1_last:"=+%
 @set _args1=
 
-@CALL "%~dp0..\condabin\conda_hook.bat"
-
-@if "%_args1_first%"=="+" if NOT "%_args1_last%"=="+" (
-    @CALL conda.bat activate
-    @GOTO :End
-)
```

Comment:
I believe this if statement block is needed to deal with users with spaces in their path, see #6825

Reply:
[This comment](https://github.com/conda/conda/pull/6825#discussion_r165779154) in particular is helpful.

Reply:
Thanks, @kalefranz had the same concern.  I think I've addressed it with the latest push.